internal/services/network/network_manager_verifier_workspace_reachability_analysis_intent_resource_test.go

```diff
@@ -0,0 +1,222 @@
+	r := ManagerVerifierWorkspaceReachabilityAnalysisIntentResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (r ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := reachabilityanalysisintents.ParseReachabilityAnalysisIntentID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := client.Network.ReachabilityAnalysisIntents.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving %s: %+v", *id, err)
+	}
+
+	return pointer.To(resp.Model != nil), nil
+}
+
+func (r ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%[1]s
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_network_manager_verifier_workspace_reachability_analysis_intent" "test" {
+  name                    = "acctest-vw-%[2]d"
+  verifier_workspace_id   = azurerm_network_manager_verifier_workspace.test.id
+  source_resource_id      = azurerm_linux_virtual_machine.test.id
+  destination_resource_id = azurerm_linux_virtual_machine.test.id
+  ip_traffic {
+    source_ips        = ["10.0.2.0"]
+    source_ports      = ["80"]
+    destination_ips   = ["10.0.2.0"]
+    destination_ports = ["*"]
+    protocols         = ["Any"]
+  }
+}
+`, r.template(data), data.RandomInteger)
+}
+
+func (r ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) requiresImport(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_network_manager_verifier_workspace_reachability_analysis_intent" "import" {
+  name                    = azurerm_network_manager_verifier_workspace_reachability_analysis_intent.test.name
+  verifier_workspace_id   = azurerm_network_manager_verifier_workspace.test.id
+  source_resource_id      = azurerm_linux_virtual_machine.test.id
+  destination_resource_id = azurerm_linux_virtual_machine.test.id
+  ip_traffic {
+    source_ips        = ["10.0.2.0"]
+    source_ports      = ["80"]
+    destination_ips   = ["10.0.2.0"]
+    destination_ports = ["*"]
+    protocols         = ["Any"]
+  }
+}
+`, r.basic(data))
+}
+
+func (r ManagerVerifierWorkspaceReachabilityAnalysisIntentResource) complete(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%[1]s
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_network_manager_verifier_workspace_reachability_analysis_intent" "test" {
+  name                    = "acctest-vw-%[2]d"
+  verifier_workspace_id   = azurerm_network_manager_verifier_workspace.test.id
+  source_resource_id      = azurerm_linux_virtual_machine.test.id
+  destination_resource_id = azurerm_linux_virtual_machine.test.id
+  description             = "test"
+  ip_traffic {
```

Comment:
since we are allowing multiple we should have multiple ip_traffic blocks here?

Reply:
only one block is allowed